keyboards/viktus/sp_mini/keymaps/peott-fr/keymap.c

```diff
@@ -0,0 +1,114 @@
+  * it under the terms of the GNU General Public License as published by
+  * the Free Software Foundation, either version 2 of the License, or
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#include QMK_KEYBOARD_H
+
+// Layers by name
+enum custom_layers {
+    _BASE,
+    _LHAND,
+    _RHAND,
+    _NUM
+};
+
+// Layers defined by function
+#define LHAND LT(_LHAND,KC_SPC)
+#define RHAND LT(_RHAND,KC_BSPC)
+#define NUM TG(3)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+        [_BASE] = LAYOUT_all(
+			KC_MPLY, KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, 					 	KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSLS,
+			KC_HOME, KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, 					 	KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, NUM,
+			KC_PGUP, KC_LCTL, KC_A, KC_S, KC_D, KC_F, KC_G,					 	KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_DEL,
+			KC_PGDN, KC_LSPO, KC_Z, KC_X, KC_C, KC_V, KC_B, 				 	KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_UP,
+			KC_END, LCTL_T(KC_MPRV), KC_LGUI, LALT_T(KC_MNXT), KC_NO, LHAND, 	RHAND, KC_RALT, KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+			),
+        [_LHAND] = LAYOUT_all(
+			RESET, KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, 			   	KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_PSCR,
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS,   	RGB_TOG, RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, KC_TRNS, KC_TRNS,
+			KC_TRNS, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, 	KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_ENT, KC_TRNS,
+			KC_TRNS, KC_LSFT, KC_CALC, KC_MYCM, KC_TRNS, KC_ENT, KC_BSPC, 	KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_RSPC, KC_TRNS,
+			KC_TRNS, KC_LCTL, KC_TRNS, KC_LALT, KC_TRNS, KC_TRNS,          	KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+			),
+        [_RHAND] = LAYOUT_all(
+			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, 	KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, 	KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, 	KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, 	KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,           KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO
+			),
+        [_NUM] = LAYOUT_all(
+			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, 	KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS, KC_NO, KC_NO, KC_NO, KC_F13,
+			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, 	KC_P7, KC_P8, KC_P9, KC_PPLS, KC_NO, KC_NO, KC_NO, KC_NO,
```

Comment:
```suggestion
			KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, 	KC_P7, KC_P8, KC_P9, KC_PPLS, KC_NO, KC_NO, KC_NO, NUM,
```
You need the toggle key on the target layer or you wont be able to get back.